<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Bgp</class>
 <widget class="QWidget" name="Bgp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">gridline-color: rgb(0, 0, 0);
selection-color: rgb(0, 170, 255);
selection-background-color: rgb(66, 66, 66);
background-color: rgb(70, 70, 70);
alternate-background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QPushButton" name="exitBtn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>260</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 255);
background-color: rgb(66, 66, 66);
border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
  <widget class="QPushButton" name="configBtn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>260</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 255);
background-color: rgb(66, 66, 66);
border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Configure</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearBtn">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>260</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 255);
background-color: rgb(66, 66, 66);
border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="routeProcBox">
   <property name="geometry">
    <rect>
     <x>108</x>
     <y>10</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);
border-color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color :rgb(0, 170, 255)</string>
   </property>
   <property name="text">
    <string>Routing Process</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="neighborBox1">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>50</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);
border-color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>52</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color :rgb(0, 170, 255)</string>
   </property>
   <property name="text">
    <string>Neighbor</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="neighborBox2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);
border-color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="neighborBox4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);
border-color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="neighborBox3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);
border-color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>90</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color :rgb(0, 170, 255)</string>
   </property>
   <property name="text">
    <string>AS Number</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="asNumBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>88</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);
border-color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="netBox3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);
border-color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="netBox1">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);
border-color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>42</x>
     <y>132</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color :rgb(0, 170, 255)</string>
   </property>
   <property name="text">
    <string>Network</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="netBox4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);
border-color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="netBox2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);
border-color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="maskBox2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);
border-color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="maskBox3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>170</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);
border-color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color :rgb(0, 170, 255)</string>
   </property>
   <property name="text">
    <string>Mask</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="maskBox1">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);
border-color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="maskBox4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>170</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);
border-color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="disableCheck">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>85</width>
     <height>22</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);
border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Disable</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="loopBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);
border-color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>212</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color :rgb(0, 170, 255)</string>
   </property>
   <property name="text">
    <string>Loopback</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>routeProcBox</tabstop>
  <tabstop>disableCheck</tabstop>
  <tabstop>neighborBox1</tabstop>
  <tabstop>neighborBox2</tabstop>
  <tabstop>neighborBox3</tabstop>
  <tabstop>neighborBox4</tabstop>
  <tabstop>asNumBox</tabstop>
  <tabstop>netBox1</tabstop>
  <tabstop>netBox2</tabstop>
  <tabstop>netBox3</tabstop>
  <tabstop>netBox4</tabstop>
  <tabstop>maskBox1</tabstop>
  <tabstop>maskBox2</tabstop>
  <tabstop>maskBox3</tabstop>
  <tabstop>maskBox4</tabstop>
  <tabstop>loopBox</tabstop>
  <tabstop>clearBtn</tabstop>
  <tabstop>configBtn</tabstop>
  <tabstop>exitBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
